.archon/commands/defaults/archon-create-pr.md

```
@@ -7,9 +7,12 @@ argument-hint: [base-branch] (default: auto-detected from config or repo)
 
 **Base branch override**: $ARGUMENTS
 **Default base branch**: $BASE_BRANCH
+**Forge**: $FORGE_TYPE
 
 > If a base branch was provided as argument above, use it for `--base`. Otherwise use the default base branch.
 
```

markdownlint-cli2 (0.22.0)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)